index.html

```diff
@@ -0,0 +1,932 @@
+        <p class="latest-box-text underline change-color-red">
+          6 things I learned about the Meizu 17 Pro's design<br>
+          <span class="latest-news-author change-color-black">
+            Callum Booth -
+          </span>
+          <span class="latest-news-time">6 hours ago</span>
+        </p>
+        </a>
+      </div>
+
+      <div class="latest-news-box">
+        <a class="a-tags-latest-news" href="https://thenextweb.com/offers/2020/07/30/this-invaluable-design-and-marketing-tool-can-keep-you-from-wasting-hours-building-mock-websites/">
+        <img src="img/latest-news-6.jpg" alt="Latest news 6" class="latest-news-img">
+        <p class="latest-box-text underline change-color-red">
+          This invaluable design and marketing tool can keep you from wasting hours building mock websites <br>
+          <span class="latest-news-author change-color-black">
+            TNW Deals -
+          </span>
+          <span class="latest-news-time">7 hours ago</span>
+        </p>
+        </a>
+      </div>
+
+      <div class="latest-news-box">
+        <a class="a-tags-latest-news" href="https://thenextweb.com/apps/2020/07/30/whatsapp-mute-chats-always/">
+        <img src="img/latest-news-7.png" alt="Latest news 7" class="latest-news-img">
+        <p class="latest-box-text underline change-color-red">
+          WhatsApp might soon let you mute annoying chats forever <br>
+          <span class="latest-news-author change-color-black">
+            Mix -
+          </span>
+          <span class="latest-news-time">8 hours ago</span>
+        </p>
+        </a>
+      </div>
+
+      <div class="latest-news-box">
+        <a class="a-tags-latest-news" href="https://thenextweb.com/dd/2020/07/30/a-guide-to-testing-hooks-for-avid-react-developers/">
+        <img src="img/latest-news-8.png" alt="Latest news 8" class="latest-news-img">
+        <p class="latest-box-text underline change-color-red">
+          A guide to testing Hooks for avid React developers <br>
+          <span class="latest-news-author change-color-black">
+            Avi Aryan -
+          </span>
+          <span class="latest-news-time">8 hours ago</span>
+        </p>
+        </a>
+      </div>
+    </div>
+
+  </section>
+
+  <section class="latest-funding">
+    <div class="latest-funding-title-box">
+
+      <a class="titles-decoration" href="https://index.co/investments?utm_campaign=funding_block&utm_medium=referral&utm_source=thenextweb.com">
+      <h2 class="latest-titles titles change-color-black">
+        Latest funding rounds on Index.co >
+      </h2>
+      </a>
+
+      <div class="latest-funding-li">
+        <ul class="latest-funding-li">
+          <li class="latest-funding-item">
+            Add info &nbsp; |
+          </li>
+          <li class="latest-funding-item">
+            Get updates by email &nbsp; |
+          </li>
+          <li class="latest-funding-item">
+            See more
+          </li>
+        </ul>
```

Comment:
#### In mobile view

The order of the list items is different than the original. Also, remove the pipe (`|`) character after each of them.

| Source Page	| Clone Page |
|:---:|:---:|
|![Annotation 2020-08-04 113035](https://user-images.githubusercontent.com/57726348/89258698-47454c00-d646-11ea-9c2e-d0fe10b7bb3d.jpg)|![Annotation 2020-08-04 113007](https://user-images.githubusercontent.com/57726348/89258709-4d3b2d00-d646-11ea-80c7-edf0f7840a26.jpg)|